lib/db.js

```diff
@@ -419,19 +461,38 @@ module.exports = (
 
   DB.prototype.updateSessionToken = function (token, userAgentString) {
     log.trace({ op: 'DB.updateSessionToken', uid: token && token.uid })
-
-    if (! token.update(userAgentString)) {
+    if (! token.update(userAgentString) || ! this.redis) {
       return P.resolve()
     }
 
-    return this.pool.post('/sessionToken/' + token.id + '/update', {
+    const uid = token.uid
+    const newToken = [{
+      tokenId: token.id,
+      uid: uid,
       uaBrowser: token.uaBrowser,
       uaBrowserVersion: token.uaBrowserVersion,
       uaOS: token.uaOS,
       uaOSVersion: token.uaOSVersion,
       uaDeviceType: token.uaDeviceType,
-      lastAccessTime: token.lastAccessTime
-    })
+      lastAccessTime: token.lastAccessTime,
+      createdAt: token.createdAt
+    }]
+    let sessionTokens = []
+    if (features.isLastAccessTimeEnabledForUser(uid)) {
```

Comment:
The body of the function is
 ```javascript
return lastAccessTimeUpdates.enabled && (
        isSampledUser(lastAccessTimeUpdates.sampleRate, uid, 'lastAccessTimeUpdates') ||
        lastAccessTimeUpdates.enabledEmailAddresses.test(email)
```
so it'll work without using the email. There's a call in db.devices which does use email.